docs/source/devel/design.rst

```diff
@@ -1,19 +1,214 @@
-- `repository-service-tuf-worker <https://repository-service-tuf.readthedocs.io/projects/rstuf-worker/>`_
-- `repository-service-tuf-cli <https://repository-service-tuf.readthedocs.io/projects/rstuf-cli/>`_
+     - RSTUF is language agnostic, making any language use/integrate easily.
+
+   * RSTUF is process agnostic.
+
+     - Adding/removing artifacts goes with the existing releasing/publishing
+       process.
+
+   * RSTUF focuses on scalability and Metadata consistency.
+
+   * RSTUF guides users on TUF processes.
+
+
+The architecture design principles:
+###################################
+
+The Repository Service for TUF (RSTUF) has two services
+(``repository-service-tuf-api``, ``repository-service-tuf-worker``) and one
+command line tool (``repository-service-cli``), as described in
+:ref:`guide/general/introduction:RSTUF Components`.
+
+.. note::
+    Other services can be added as optional or required
+
+RSTUF also requires some third-party services described in
+:ref:`guide/deployment/planning/deployment:Required Infrastructure Services`.
+
+.. image:: /_static/2_1_rstuf.png
+
+The below definitions allow RSTUF API and RSTUF Worker scalability.
+
+RSTUF is Asynchronous
+=====================
+
+    * Every API request is a task.
+    * RSTUF centralizes all tasks in the Message :ref:`devel/design:Message Queue`.
+    * RSTUF stores all task results in the :ref:`devel/design:Backend Result`.
+    * :ref:`devel/design:Repository Service TUF API` **publishes** RSTUF tasks.
+    * :ref:`devel/design:Repository Service TUF Worker` **consumes** RSTUF tasks.
+
+RSTUF Repository Settings/Configuration
+=======================================
+
+    * :ref:`guide/general/introduction:RSTUF Components` are RSTUF
+      **stateless** about the Repository Settings, relying on
+      :ref:`devel/design:RSTUF settings`.
+
+    * :ref:`devel/design:RSTUF settings` stores in
+      :ref:`devel/design:Redis` using `Dynaconf <https://www.dynaconf.com>`_.
+
+    * :ref:`devel/design:Repository Service TUF Worker` **writes** settings
+    * :ref:`devel/design:Repository Service TUF API` **reads** settings.
+
+        .. note::
+          A unique exception is when a bootstrap process fails and the API
+          **writes** it back to ``None``.
+
+    .. list-table:: RSTUF reserved settings/configuration
+        :header-rows: 1
+        :widths: 30 30 40
+
+        * - Key
+          - Value(s)
+          - Description
+        * - ``BOOTSTRAP``
+          - | ``None``
+            | ``<task id>``
+            | ``pre-<task id>``
+            | ``sign-<task id>``
+          - | RSTUF bootstrap state
+            | ``None``: No bootstrap
+            | ``<task id>``: Finished
+            | ``pre-<task id>``: Initial process
+            | ``sign-<task id>``: Signing process
+        * - ``<ROLE NAME>_EXPIRATION``
+          - | ``int``
+            | days
+          - | Role Metadata expiration setting
```

Comment:
I don't see why you explain the values inside the `values` column when there is a specific `description` column for that.
I mean these four settings precisly:
<img width="353" alt="image" src="https://github.com/repository-service-tuf/repository-service-tuf/assets/16246778/c1e59f28-c4ee-4a48-91ea-b7ee9ecdacce">

I have suggested below how to fix them:
```suggestion
          - | Role Metadata expiration policy in days
```